CoreFoundation/Base.subproj/CFPriv.h

```diff
@@ -220,8 +222,10 @@ typedef CF_OPTIONS(CFOptionFlags, CFSearchPathDomainMask) {
     kCFAllDomainsMask = 0x0ffff	/* all domains: all of the above and more, future items */
 };
 
+#if DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED
```

Comment:
Since this is a private header, the macro must be the `TARGET_OS` style, as those are defined in a public header but `DEPLOYMENT_TARGET` are internal-only.

We're going to eventually remove all of these in favor of `TARGET_OS` everywhere but that will come later.

Reply:
Nice!  Oooh, thank you so much for that explanation!  I never understood the difference between the `DEPLOYMENT_*` and `TARGET_OS_*` macros.  I'll change this.